python/setup.py

```diff
@@ -92,22 +92,30 @@ def run(self):
             pass
         _sdist.run(self)
 
-
 class VWBuildExt(_build_ext):
     """Build pylibvw.so and install it as a python extension """
     def build_extension(self, ext):
         prep()
-        env = environ
-        env['PYTHON_VERSION'] = '{v[0]}.{v[1]}'.format(v=sys.version_info)
-        env['PYTHON_LIBS'] = '-l {}'.format(find_boost())
-        subprocess.check_call(['make', 'python'], cwd=path.join(here, 'src'), env=env)
         target_dir = path.dirname(self.get_ext_fullpath(ext.name))
         if not path.isdir(target_dir):
             makedirs(target_dir)
-        ext_suffix = 'so' if not system == 'Cygwin' else 'dll'
-        copy(path.join(here, 'src', 'python', '{name}.{suffix}'.format(name=ext.name, suffix=ext_suffix)),
-             self.get_ext_fullpath(ext.name))
-
+        if system == 'Windows':
```

Comment:
just meant the final copy of the pyvw artifact is the same, so we could do something like:

```
if system == 'Windows':
  ext_dir = path.join(here, 'bin')
  ext_suffix = 'dll'
else:
   env = environ
   env['PYTHON_VERSION'] = '{v[0]}.{v[1]}'.format(v=sys.version_info)
   env['PYTHON_LIBS'] = '-l {}'.format(find_boost())
   subprocess.check_call(['make', 'python'], cwd=path.join(here, 'src'), env=env)
   ext_dir = path.join(here, 'src', 'python')
   ext_suffix = 'so' if not system == 'Cygwin' else 'dll'

copy(path.join(ext_path, '{name}.{suffix}'.format(name=ext.name, suffix=ext_suffix)), self.get_ext_fullpath(ext.name))
```

but it's not really a big deal